espnet/nets/pytorch_backend/transducer/search.py

```diff
@@ -0,0 +1,654 @@
+"""Search algorithms for transducer models."""
```

Comment:
ditto.
maybe you can move this as https://github.com/espnet/espnet/blob/master/espnet/nets/beam_search_transducer.py ???

Again, @ShigekiKarita, what do you think?

Reply:
agree